ui/src/pageLayout/constants/navItemActivation.ts

```diff
@@ -0,0 +1 @@
+export const HOMEPAGE_PATHNAME = 'me'
```

Comment:
can you drop this into this file:
https://github.com/influxdata/influxdb/blob/master/ui/src/shared/constants/index.ts